rank/scripts/checkTask.ts

```diff
@@ -2,25 +2,58 @@ import { info, success } from './utils'
 
 import cliProgress from 'cli-progress'
 import { getRankClient } from '../services/elasticsearch'
+import prompts from 'prompts'
+import yargs from 'yargs'
 
 async function go() {
   const client = getRankClient()
-  const reindexTasks = await client.cat.tasks({
-    actions: [
-      'indices:data/write/reindex',
-      'indices:data/write/update/byquery',
-    ],
-  })
-
-  // the task response comes back as a whitespace delimeted string like:
-  //   action  task_id  parent_task_id  type  start_time  timestamp  running_time  ip  node
-  // from which we extract the task_id
-  // see https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-tasks.html
-  const task_id = reindexTasks.body.split(' ')[1]
+  let task_id
+  const args = yargs(process.argv)
+    .options({
+      task: {
+        type: 'string',
+        description: 'The task to check',
+      },
+    })
+    .exitProcess(false)
+    .parseSync()
+
+  if (args.task) {
+    task_id = args.task
+  } else {
+    const tasks = await client.cat
+      .tasks()
+      .then((res) => res.body)
+      // the task response comes back as a list of whitespace delimeted string like:
+      //   action  task_id  parent_task_id  type  start_time  timestamp  running_time  ip  node
+      // from which we extract the task_id
+      // see https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-tasks.html
+      .then((body) => body.split('\n'))
+      .then((lines) =>
+        lines.map((line) => {
+          const [action, task_id] = line.split(' ')
+          return { action, task_id }
+        })
+      )
+      .then((tasks) =>
+        tasks.filter((task) => task.task_id && task.task_id !== '')
+      )
+
+    task_id = await prompts({
```

Comment:
Minor: I don't know how many tasks you typically expect here; in scripts like this I sometimes have an optimisation:

```
if tasks.length() == 1 {
  task = tasks[0]
} else {
  task = ask_user_to_pick_from(tasks)
}
```